lib/kernel/matchers/languages.ml

```diff
@@ -497,6 +497,23 @@ module Php = struct
   end
 end
 
+module HCL = struct
+  module Info = struct
+    let name = "HashiCorp Configuration Language"
+    let extensions = [".hcl"]
```

Comment:
@rvantonder – a common extension for Terraform code written in HCL is `.tf`

https://sourcegraph.com/search?q=file:%5C.tf%24&patternType=literal

i can see some `.hcl` files but I'm not sure if they are even correct: https://sourcegraph.com/search?q=file:%5C.hcl%24&patternType=literal